.packit.yaml

```diff
@@ -0,0 +1,39 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+---
+require:
+  label:
+    absent:
+      - no-ci-builds
+
+actions:
+  create-archive:
+    - bash -c "tito build --tgz --test -o ."
+    - bash -c "ls -1t ./*.tar.gz | head -n 1"
+  get-current-version:
+    - bash -c "grep ^Version \"$PACKIT_CONFIG_PACKAGE_NAME.spec\" | awk '{ print $2 }'"
+
+packages:
+  fedora-distro-aliases:
+    specfile_path: fedora-distro-aliases.spec
+    downstream_package_name: fedora-distro-aliases
+    upstream_tag_template: fedora-distro-aliases-{version}
+srpm_build_deps:
+  - tito
+  - git
+
+jobs:
+  - <<: &pr_build
+      job: copr_build
+      trigger: pull_request
+      packages:
+        - fedora-distro-aliases
+      targets:
+        - fedora-all
+
+  - <<: *pr_build
+    trigger: commit
+    owner: "@copr"
+    project: "copr"
```

Comment:
I guess this is a leftover from copy-pasting. 

Currently, I have this Copr project 
https://copr.fedorainfracloud.org/coprs/frostyx/fedora-distro-aliases/ 

but we should probably create a new one in 
https://copr.fedorainfracloud.org/groups/g/rpm-software-management/coprs/ 
I don't have permissions though.

Reply:
No a c&p error, my mistake - I thought this lives in the `fedora-copr` namespace.